gdx/src/com/badlogic/gdx/utils/Align.java

```diff
@@ -42,9 +42,7 @@ static public final boolean isTop (int align) {
 		return (align & top) != 0;
 	}
 
-	static public final boolean isBottom (int align) {
-		return (align & bottom) != 0;
-	}
+	static public final boolean isBottom (int align) { return (align & bottom) != 0; }
```

Comment:
I'd discourage pushing any changes to other classes unless there is something in them, however minor, that needs fixing. Spotless will revert this anyway (run the `gdx:spotlessApply` Gradle task if you want the CI/CD checks to pass) but it won't revert the removal of trailing whitespace in the copyright header, if I'm not mistaken. I don't know whether there's a consensus on if it should have the trailing whitespace or not, but I wouldn't have that as the sole change in any case (clutters the commit history for no gain).

Reply:
Sorry, I didn't mean to commit these changes and it wasn't intentional. I don't master Git yet.

Reply:
Don't worry about it too much. Sometimes I limit what I add to reduce the chances of accidentally including other stuff, e.g. `git add gdx/test` instead of `git add .`, sometimes I don't.